File: android/app/src/main/java/app/tourism/data/dto/place/UserDto.kt
package app.tourism.data.dto.place

import app.tourism.domain.models.details.User


data class UserDto(
    val id: Long,
    val avatar: String,
    val country: String,
    val full_name: String,
) {
    fun toUser() = User(
        id = id,
        name = full_name,
        countryCodeName = country,
        pfpUrl = avatar,
    )
}

File: iphone/Maps/Tourism/Data/Network/DTO/Details/UserDTO.swift
import Foundation

struct UserDTO: Codable {
  let id: Int64
  let avatar: String
  let country: String
  let fullName: String
  
  func toUser() -> User {
    return User(
      id: id,
      name: fullName,
      pfpUrl: avatar,
      countryCodeName: country
    )
  }
}
